### PR TITLE
Add basic URL fetching support

### PR DESCRIPTION
**Pull request by pedropedruzzi**
What do you think of these feature sketch?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,8 +89,6 @@
 
     // convert to CSV, make available
     doCSV(json);
-
-    return true;
```
Comment (konklone) [inline index.html:76]: Why remove `return true`?

**Comment by konklone** [comment]
I've been wanting to add this feature for a while! So I'm in favor of it.

I don't know about pasting the URL into the giant textarea though. I feel like that will lead to confusing instructions, and weirder code.

How about adding a separate text field above the textarea for a URL? If you add it, and make the code work, I can take care of styling it and making it look nice before it's deployed.

**Comment by pedropedruzzi** [comment]
I agree about having a separate field.

I was wondering if (and how) we could support appending rows to the table by pasting or fetching new JSON data (with nearly the same set of attributes). Do you think it would be useful to many users?

How about some new buttons like: "clear", "clear but keep rows". Alternative: a checkbox "Append data to the table (instead of replacing)".

**Comment by konklone** [comment]
(Just a note that I'm traveling now, and won't be able to give this its due diligence until next week.)

> I was wondering if (and how) we could support appending rows to the table by pasting or fetching new JSON data (with nearly the same set of attributes). Do you think it would be useful to many users?
> 
> How about some new buttons like: "clear", "clear but keep rows". Alternative: a checkbox "Append data to the table (instead of replacing)".

This sounds useful, but also maybe too complicated. I think the first step is to just get a field for remotely importing URLs, and go from there. I really want to keep the site very friendly and unintimidating for people who are totally new to JSON. Even the few controls that are there in the UI right now actually took a lot of work and iteration until I was satisfied that it was easy enough to use.

**Comment by jqnatividad** [comment]
Any chance this feature will be revisited?  That is, just the original function @pedrox proposed, with just a separate textfield for the URL?

**Comment by konklone** [comment]
@jqnatividad If it's simple enough code, with a simple enough user experience, sure. What's your use case?

**Pull request closed without merging**